docs/registry/entities/workflow.md

```diff
@@ -0,0 +1,181 @@
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/entity_namespace.md.j2 -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+# Workflow
+
+## Workflow
+
+**Status:** ![Development](https://img.shields.io/badge/-development-blue)
+
+**type:** `workflow`
+
+**Description:** 
+
+**Identifying Attributes:**
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`workflow.id`](/docs/registry/attributes/workflow.md) | string | Unique ID for this workflow which can be used to correlate multiple executions of this workflow. | `5c6e9a4b-69ab-499f-b4a7-4cf5c8720d66` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
+
+**Descriptive Attributes:**
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`workflow.name`](/docs/registry/attributes/workflow.md) | string | Name of the workflow | `Build and deploy application` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
+| [`workflow.type`](/docs/registry/attributes/workflow.md) | string | Type of the workflow | `DSL` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
+| [`workflow.url.full`](/docs/registry/attributes/workflow.md) | string | The [URL](https://wikipedia.org/wiki/URL) of the workflow, providing the complete address in order to locate and identify the workflow. | `https://github.com/open-telemetry/semantic-conventions/actions/runs/9753949763?pr=1075` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
+| [`workflow.version`](/docs/registry/attributes/workflow.md) | string | Version of the workflow | `2.1` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
+
+## Workflow Execution
```

Comment:
Unless there is some sort of historic reasoning for this, I'd opt for deciding between the terms `execution` and `run` and using one of them for both workflows and tasks. Imho, adding tasks as an inherent property of workflows is good, but the phrasing should be consistent.

I'd go for `workflow.run.*` and `task.run.*` for sake of simplicity.

Reply:
For some history: in cicd we had a discussion about the naming of runs and we landed on run in part because it aligned well with the terminology of CDEvents.
cf. https://github.com/open-telemetry/semantic-conventions/pull/1075#discussion_r1615339395